_pages/01-applicants.md

```diff
@@ -1,23 +1,25 @@
 ---
 layout: applicant
 title: Information for Applicants
-description: <b>Due to COVID-19, there will be no in-person event in 2020. 
-    A virtual event around August is being discussed.
-    Applications will open soon.
-    Please check back in early May.</b><br><br>
+description: <b>Due to COVID-19, OceanHackWeek (OHW) 2020 will take place as a virtual, online event.
+    <a href="applicant-info.html">Applications</a> close on July 13, 2020. 
 
     In OceanHackWeek we will explore the intersection of data science 
     and oceanography through tutorials and hands-on “hacking” projects. 
     To best benefit from the program, participants are expected to have some experience with Python programming and data analysis.
 permalink: applicant-info.html
 image: ohw18-hacking2.JPG
-
 ---
 
-**Due to COVID-19, there will be no in-person event in 2020. A virtual event around August is 
-being discussed. Applications will open soon. Please check back in early May.**
+OceanHackWeek 2020 is a small, hands-on, interactive virtual workshop focused on data science and oceanography that will be held during **August 10-14, 2020**. Join us for five days of tutorials, data exploration, software development and community networking!
+
+**To apply, please [fill out this form](https://form.jotform.com/201774238098159) by July 13, 2020.**  
+Accepted applicants will be notified no later than July 22, 2020.  
+We expect to accommodate approximately 50 participants.
 
-In Oceanhackweek we will explore the intersection of data science and oceanography through tutorials and hands-on “hacking” projects. In tutorials, we will learn data science tools, cloud computing, visualization, and a suite of software assets to interact with data sets of complex temporal-spatial structures or high volume. In project sessions, we will immediately put these skills to use by implementing research, computation, or visualization ideas in a group setting. To best benefit from the program, participant are expected to have some experience with Python programming and data analysis.
+**Formal daily activities will take place over a period of up to 3 hours per day, tentatively 10am-1pm US PDT / 1-4pm US EDT / 17:00-20:00 UTC.** Formal daily activities includes tutorials and project check-in with the entire group, but excludes actual project time. The time period will be finalized based on the geographical distribution of participants.
```

Comment:
"The time period will be finalized based on the geographical distribution of participants."

This is vague and may discourage people from applying and/or we may accept someone from a time-zone that won't be able to make it later.

I'd rather have a time slot that we know the instructors will be able to teach. CarpentryCon set proxies to repeat the record presentations for other time-zones. We don't have that but we can try to publish the recorded lesson ASAP and encourage people to watch them and still participate in the other parts of the event that are asynchronous, like the project and help desks.

Reply:
I think you're suggesting that we can just drop the last sentence, right? I'm good with that, so I'll do it right away before merging.

Reply:
I've also removed the sentence from the application form text.

Reply:
> I think you're suggesting that we can just drop the last sentence, right?

Yep.